internal/infrastructure/mongo/asset_test.go

```diff
@@ -0,0 +1,108 @@
+package mongo
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/reearth/reearth-backend/internal/infrastructure/mongo/mongodoc"
+	"github.com/reearth/reearth-backend/pkg/asset"
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/x/mongo/driver/uuid"
+)
+
+func TestFindByID(t *testing.T) {
+	// Skip unit testing if "REEARTH_DB" is not configured
+	// See details: https://github.com/reearth/reearth/issues/273
+	db := os.Getenv("REEARTH_DB")
+	if db == "" {
+		return
+	}
+
+	type a struct {
```

Comment:
Note 📖 

I locally named `a` but if you have preference, please let me know.
(Also, we can rename the `asset` in the `import` to other name )

Reply:
Ah, I didn't noticed the `New()` function... Thank you for pointing it out! 👉 Updated 👉 https://github.com/reearth/reearth-backend/pull/139/commits/06278f8c9f22c5237f9574d9f23ef4d8e00372aa